### ISSUE TITLE
vuejs fw demo is broken

### ISSUE THREAD

**Issue by mh-cbon** [material-components/material-components-web#364]
** PLEASE READ THIS BEFORE FILING AN ISSUE **

### What MDC-Web Version are you using?

> https://github.com/material-components/material-components-web/commit/f8d72ba72b2fb47a2027a09d81f4702581875d90
v0.5.0 ?

### What browser(s) is this bug affecting?

> all

### What OS are you using?

> fedora

### What are the steps to reproduce the bug?

- dl zip tarball of the github, extract vue js demo to a folder, npm i, npm start

it will complain with :

```sh
 $ npm start

> v-mdc-example@0.1.0 start /home/mh-cbon/gow/src/github.com/mh-cbon/backup/web2
> node build/dev-server.js

Listening at http://localhost:8080

webpack: wait until bundle finished: /index.html
webpack built 525205a7d73bdb18d5ff in 11320ms
Hash: 525205a7d73bdb18d5ff
Version: webpack 1.14.0
Time: 11320ms
 Asset     Size  Chunks       Chunk Names
app.js  2.19 MB       0       app

ERROR in ./~/vue-loader/lib/template-compiler.js?id=data-v-5138956e!./~/vue-loader/lib/selector.js?type=template&index=0!./src/v-mdc-snackbar/Snackbar.vue
template syntax error tag <div> has no matching end tag.

ERROR in ./~/vue-loader/lib/template-compiler.js?id=data-v-0665e9ee!./~/vue-loader/lib/selector.js?type=template&index=0!./src/v-mdc-checkbox/Checkbox.vue
template syntax error tag <div> has no matching end tag.
```

### What is the expected behavior?

The demo just works.

### What is the actual behavior?

bug.

### Any other information you believe would be useful?

nop

**Comment by guillaumevincent** [comment]
@mh-cbon I created a PR https://github.com/material-components/material-components-web/pull/365 that fix this one

**Comment by traviskaufman** [comment]
closed via #365 Thanks @guillaumevincent !
